client/assets/js/maptest.js

```diff
@@ -12,6 +12,14 @@ var target = {
     threshold: 8
```

Comment:
We'll need to get a few test sites available for people. I'll work on having it iterate over an array of them to check on my next pass. Then we wont need to swap these as much.

Reply:
By test sites I mean testing location datas (places).